index.html

```diff
@@ -38,5 +38,161 @@ <h2>
+                <ul>
+                  <li class="html">html</li>
+                  <li class="boostrap">boostrap</li>
+                  <li class="ruby">ruby</li>
+                </ul>
+                <button>See Project</button>
+            </div>
+            <div class="card-works">
+              <h1>Profesional Art Printing Data</h1>
+              <p>
+                  A daily selection of privately personalized reads; 
+                  no accounts or sign-ups required. 
+                  has been the industry's standard
+                </p>
+                <ul>
+                  <li class="html">html</li>
+                  <li class="boostrap">boostrap</li>
+                  <li class="ruby">ruby</li>
+                </ul>
+                <button>See Project</button>
+            </div>
+            <div class="card-works">
+              <h1>Profesional Art Printing Data</h1>
+              <p>
+                  A daily selection of privately personalized reads; 
+                  no accounts or sign-ups required. 
+                  has been the industry's standard
+                </p>
+                <ul>
+                  <li class="html">html</li>
+                  <li class="boostrap">boostrap</li>
+                  <li class="ruby">ruby</li>
+                </ul>
+                <button>See Project</button>
+            </div>
+            <div class="card-works">
+              <h1>Profesional Art Printing Data</h1>
+              <p>
+                  A daily selection of privately personalized reads; 
+                  no accounts or sign-ups required. 
+                  has been the industry's standard
+                </p>
+                <ul>
+                  <li class="html">html</li>
+                  <li class="boostrap">boostrap</li>
+                  <li class="ruby">ruby</li>
+                </ul>
+                <button>See Project</button>
+            </div>
+            <div class="card-works">
+              <h1>Profesional Art Printing Data</h1>
+              <p>
+                  A daily selection of privately personalized reads; 
+                  no accounts or sign-ups required. 
+                  has been the industry's standard
+                </p>
+                <ul>
+                  <li class="html">html</li>
+                  <li class="boostrap">boostrap</li>
+                  <li class="ruby">ruby</li>
+                </ul>
+                <button>See Project</button>
+            </div>
+            <div class="about-me">
+              <h1>About me</h1>
+              <p>
+                  Hello I’m a software developer! I can help you build a product , 
+                  feature or website Look through some of my work and experience! 
+                  If you like what you see and have a project you need coded, 
+                  don’t hestiate to contact me.
+              </p>
+              <button>Get My Resume</button>
+              <hr>
```

Comment:
For better semantics, I think it would be better to have the about-me section in a different `section` other than the works section. I recommend that you do so.